src/api/ledger/parse/utils.js

```diff
@@ -1,10 +1,25 @@
 /* @flow */
 'use strict';
 const _ = require('lodash');
+const BigNumber = require('bignumber.js');
 const transactionParser = require('ripple-lib-transactionparser');
 const toTimestamp = require('../../../core/utils').toTimestamp;
 const utils = require('../utils');
 
+/*:: type XRPAmount = {currency: string, value: string} */
```

Comment:
Maybe, I haven't had time to try it yet: https://github.com/facebook/flow/issues/267